### DISCUSSION TITLE
Markdown: Indented description of task list item

### DISCUSSION THREAD

**Discussion by wilkomanger** [requarks/wiki#6882] [Help / Questions] [answered]
Hi, perhaps I'm missing something, but when I have markdown such as:
```
- [ ] Test item
  Test description
```
I would like it to be rendered as:
- [ ] Test item
 Test description

However, on WikiJS it's rendered as (no indent):
![image](https://github.com/requarks/wiki/assets/51335189/5030abbd-2b21-4152-a4dc-6f6bf9d17930)

It this already possible and it's done differently? Or not supported yet?

**Comment by NGPixel** [comment, marked as answer]
Not supported.
